templates/SynapseExternalBucket.yaml

```diff
@@ -25,9 +35,11 @@ Parameters:
   SynapseUserName:
     Type: String
     Description: Synapse user for read-write bucket
+    AllowedPattern: ".+" # makes this parameter required
```

Comment:
When left empty on the clouformation console webpage, errors would occur after the resources were created and the Outputs were being processed made. Users are not allowed to proceed these parameters were not empty

Reply:
hmm, interesting.  Cloudformation would run and resources would get created even if you don't set  a value for SyanpaseUserName?  That's not my experience with the CLI.  it will fail without running the template at all.  This system uses the CLI

Reply:
I'll remove the regex then. I thought users might be taking this template and running it on the aws console.